examples/guestbook/README.md

```diff
@@ -323,28 +342,47 @@ if (isset($_GET['cmd']) === true) {
 } ?>
 ```
 
-To play with the service itself, find the name of a frontend, grab the external IP of that host from the [Google Cloud Console][cloud-console] or the `gcloud` tool, and visit `http://<host-ip>:8000`.
+To play with the service itself, find the name of a frontend,
+
+### A few Google Container Engine specifics for playing around with the services.
 
+- In GCE, you can grab the external IP of that host from the [Google Cloud Console][cloud-console] or the `gcloud` tool, and visit `http://<host-ip>:8000`.
 ```shell
 $ gcloud compute instances list
 ```
-
-You may need to open the firewall for port 8000 using the [console][cloud-console] or the `gcloud` tool. The following command will allow traffic from any source to instances tagged `kubernetes-minion`:
+In GCE, you also may need to open the firewall for port 8000 using the [console][cloud-console] or the `gcloud` tool. The following command will allow traffic from any source to instances tagged `kubernetes-minion`:
 
 ```shell
 $ gcloud compute firewall-rules create --allow=tcp:8000 --target-tags=kubernetes-minion kubernetes-minion-8000
 ```
 
-If you are running Kubernetes locally, you can just visit http://localhost:8000.
-For details about limiting traffic to specific sources, see the [GCE firewall documentation][gce-firewall-docs].
+For GCE details about limiting traffic to specific sources, see the [GCE firewall documentation][gce-firewall-docs].
 
 [cloud-console]: https://console.developer.google.com
 [gce-firewall-docs]: https://cloud.google.com/compute/docs/networking#firewalls
 
+In other environments, you can get the service IP from looking at the output of `kubectl get pods,services`, and modify your firewall using standard tools and services (firewalld, iptables, selinux) which you are already familar with.
+
+And of course, finally, if you are running Kubernetes locally, you can just visit http://localhost:8000.  
+
 ### Step Six: Cleanup
 
-To turn down a Kubernetes cluster:
+To turn down a Kubernetes cluster, if you ran this from source, you can use 
 
 ```shell
 $ cluster/kube-down.sh
 ```
+
+If you are in a live kubernetes cluster, you can just kill the pods, using a script such as this (obviously, read through it and make sure you understand it before running it blindly, as it will kill several pods automatically for you).
+
+```shell
+### First, kill services and controllers.
+kubectl delete -f examples/guestbook/redis-slave-controller.json
+kubectl delete -f examples/guestbook/frontend-controller.json
```

Comment:
Please use kubectl stop rc ... for stopping replication controllers in the example. This will automatically remove pods created by replication controller.